src/StorageSync/StorageSync/Interop/ManagedIdentity/ServerManagedIdentityUtils.cs

```diff
@@ -0,0 +1,375 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ServerManagedIdentityUtils.cs" company="Microsoft Corporation.">
+//   All rights reserved.
+// </copyright>
+// <summary>
+//   Utility Class for Server Managed Identity
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Kailani.Afs.ServerManagedIdentity
```

Comment:
Why the namespace is like this?

Reply:
Yes, it is copied from our msazure repo. 

Can be fixed but not a blocker. We can take this change in future PRs before it's requested for main branch